R/OsModel.R

```diff
@@ -108,6 +108,12 @@ setMethod(
 
 #' getter method for OsModel
 
+#' getter method for OsModel:
+#' @rdname priors
+#' @param object A `OsModel` object generated from function [OsModel()]
+#' @export
+
+
```

Comment:
```suggestion
#' @rdname priors
#' @export
```
is actually sufficient and better because the general `priors()` documentation is otherwise overwritten. please use same pattern for the other methods